lms/envs/aws.py

```diff
@@ -176,7 +176,11 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
I think a better approach is just to change the line above to

```python
LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL', LMS_ROOT_URL)
```

because `LMS_ROOT_URL` is already set in `common.py`, so it's never undefined up to this point - we just need to avoid changing it to `None` in the event that `LMS_ROOT_URL` does not appear in `lms.env.json`.